### ISSUE TITLE
Ink problems

### ISSUE THREAD

**Issue by Meetem** [dukelec/mb#23]
Hello, I've just got my PrinCube, and tried to test it with default images. But the ink seems pushed wrong and there are some color drops, and also yellow color is not printed fully. I've downloaded latest firmware but issue is still there. I don't know is it hardware wise or not.

![Here is an image](https://downloader.disk.yandex.ru/preview/9889191959474ec5305626af6eaf6c5110bf54ba63a74b2ad0093c34cdfa8d03/5f89dfa7/P7-seMLPjsn1jfLWsJIUZg6kjYvzYrhdg9Om42TIE9P8P0_pvxrrFfRgzVRpNR0n9VvTtqNdjfDF6tsS7z-KOA==?uid=0&filename=photo_2020-10-16_16-59-49.jpg&disposition=inline&hash=&limit=0&content_type=image%2Fjpeg&tknv=v2&owner_uid=343685984&size=1331x937)

**Comment by dukelec** [comment]
Sorry for this, please read this first:
English FAQ (Important): http://blog.d-l.io/mb-faq

**Comment by Meetem** [comment]
@dukelec yes, I've tried eraser trick, even tried some IPA to clean cartridge pins. No effect, also no effect on reconnecting cable. Both of them (even spring pins) looks pretty well and there's no visible damage. Also tried to hold transparent hood tight in hand to exclude bad contact. Problem still persists.

**Comment by dukelec** [comment]
Please try using a small blade instead of an eraser to scratch the surface of the ink cartridge contact points. If it still fails, please contact the after-sales replacement.

To be sure, the vast majority of this problem is caused by poor contact of the ink cartridges.

**Comment by manya6502** [comment]
Hello.
I got Mbrush, but it seems that yellow and magenta are the opposite. Of course, I changed the settings, but they are not reflected. How can I solve this problem?
![DSC_0066](https://user-images.githubusercontent.com/79372435/108602933-4b6d6700-73e8-11eb-8627-6df112dc06e4.JPG)
![DSC_0068](https://user-images.githubusercontent.com/79372435/108602939-51fbde80-73e8-11eb-9510-1859674ba72a.JPG)
![DSC_0067](https://user-images.githubusercontent.com/79372435/108602941-54f6cf00-73e8-11eb-9b26-799566b15605.JPG)

**Comment by dukelec** [comment]
@manya6502 Please re-download image to printer in print preview page to take effect.

**Comment by manya6502** [comment]
> @manya6502 After change color order in setting page, re-download image to printer in print preview page to take effect.

Thank you for contacting us.
By transferring it again, the color comes out correctly.
Thank you very much!!
